roles/common/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 dependencies:
+  - admin-user
```

Comment:
The common role is also run on the bastion host, and I would prefer to not include this user on that host. Can we move this role to install-ci.yml instead of being a dependency of the common role?